### PR TITLE
Updated Module Definition

### PR DESCRIPTION
**Pull request by rijnhardtkotze**

### CODE REVIEW

File: lib/root_insurance/api/quote.rb [hunk 1]
```diff
@@ -1,94 +1,96 @@
...
+        data = {
+          type:                 :root_funeral,
+          cover_amount:         opts[:cover_amount],
+          has_spouse:           opts[:has_spouse],
+          number_of_children:   opts[:number_of_children],
+          extended_family_ages: opts[:extended_family_ages]
+        }
+  
+        post(:quotes, data)
+      end
```
[the remark this answers is not in the document]
Comment (lm-pretorius) [reply lib/root_insurance/api/quote.rb:92]: Hey @rijnhardtkotze,

@houndci-bot is right - please remove the extra whitespace after the `end`

File: lib/root_insurance/api/application.rb [hunk 2]
```diff
@@ -1,44 +1,46 @@
...
+      # @return [Hash]
+      #
+      # @example
+      #   client.create_application(
+      #     policyholder_id:  "bf1ada91-eecb-4f47-9bfa-1258bb1e0055",
+      #     quote_package_id: "f4397823-db4a-4d6a-a06b-08e1a2a3172c",
+      #     monthly_premium:  50000,
+      #     serial_number:    "1234567890")
+      def create_application(policyholder_id:, quote_package_id:, monthly_premium:,
+                            serial_number: nil, spouse_id: nil, children_ids: nil, extended_famliy_ids: nil)
```
Comment (lm-pretorius) [inline lib/root_insurance/api/application.rb]: Let's make `policyholder_id` and `serial_number` line up.

File: lib/root_insurance/api/application.rb [hunk 3]
```diff
@@ -1,44 +1,46 @@
...
-      data = module_data ? data.merge(module_data) : data
+        data = module_data ? data.merge(module_data) : data
 
-      post(:applications, data)
+        post(:applications, data)
+      end
     end
   end
-end
+end
```
Comment (lm-pretorius) [inline lib/root_insurance/api/application.rb:46]: Newlines at the end of files to keep the coding gods happy.

File: lib/root_insurance/api/quote.rb [hunk 4]
```diff
@@ -1,94 +1,96 @@
...
+      # [cover_amount (Integer)]   Amount to cover, in cents. Should be between R100 000 and R5 000 000, inclusive.
+      # [cover_period (String or Symbol)]   Duration to cover: +:1_year+, +:2_years+, +:5_years+, +:10_years+, +:15_years+, +:20_years+ or +:whole_life+.
+      # [basic_income_per_month (Integer)]    Policyholder's basic monthly income, in cents.
+      # [education_status (String or Symbol)]    Policyholder’s education class: +:grade_12_no_matric+, +:grade_12_matric+, +:diploma_or_btech+, +:undergraduate_degree+ or +:professional_degree+
+      # [smoker (Boolean)]     Is the policyholder a smoker.
+      # [gender (String or Symbol)]    Gender of policyholder. Should be either +:male+ or +:female+.
+      # [age (Integer)]    Age of policyholder. Should be between 18 and 63, inclusive.
+      #
+      # == +:root_funeral+
+      # [cover_amount (Integer)]   Amount to cover, in cents. Should be between R10k and R50k, inclusive.
```
[the remark this answers is not in the document]
Comment (lm-pretorius) [reply lib/root_insurance/api/quote.rb:22]: For documentation we can ignore Hound

File: lib/root_insurance/api/quote.rb [hunk 5]
```diff
@@ -1,94 +1,96 @@
...
+      # @return [Array<Hash>]
+      #
+      # @example
+      #   client.list_gadget_models
+      #
+      def list_gadget_models
+        get('gadgets/models')
+      end
+  
+      private
```
[the remark this answers is not in the document]
Comment (lm-pretorius) [reply lib/root_insurance/api/quote.rb:57]: Please also fix this, if you don't mind.

It's actually my fault, but I think we should just follow Hound's standards, especially if more people will be contributing to this project in the future.

File: lib/root_insurance/api/quote.rb [hunk 6]
```diff
@@ -1,94 +1,96 @@
...
+      def create_gadget_quote(opts)
+        data = {
+          type:       :root_gadgets,
+          model_name: opts[:model_name]
+        }
+
```
[the remark this answers is not in the document]
Comment (lm-pretorius) [reply lib/root_insurance/api/quote.rb]: Tbh I don't know what Hound is on about here. Let's leave this for now...

File: lib/root_insurance/api/quote.rb [hunk 7]
```diff
@@ -1,94 +1,96 @@
...
+      def create_term_quote(opts)
+        data = {
+          type:             :root_term,
+          cover_amount:     opts[:cover_amount],
+          cover_period:     opts[:cover_period],
+          education_status: opts[:education_status],
+          smoker:           opts[:smoker],
+          gender:           opts[:gender],
+          age:              opts[:age],
+          basic_income_per_month: opts[:basic_income_per_month],
```
[the remark this answers is not in the document]
Comment (lm-pretorius) [reply lib/root_insurance/api/quote.rb:77]: Yup, let's remove that last comma. It is probably me again...

**Review by lm-pretorius — CHANGES_REQUESTED** [review]
Hey @rijnhardtkotze!

Thanks for the PR!
I've left a few comments, all about styling.

Although styling is not crucial, a codebase can get a bit messy if everyone contributing doesn't follow the same style guide. That's why we added @houndci-bot to this project to force us to be consistent.

A bunch of the styling issues are my fault and if you don't feel like fixing them, please don't bother - I can do it right after I merge your PR.

**Comment by rijnhardtkotze** [comment]
@glasnoster 

Updated the styling fixes. I think it is ready for merge now.

**Review by lm-pretorius — APPROVED** [review]

**Pull request merged**